dot/core/interface.go

```diff
@@ -26,6 +26,9 @@ import (
 	"github.com/ChainSafe/gossamer/lib/transaction"
 )
 
+// DEFAULT_BUFFER_SIZE buffer size for channels
+const DEFAULT_BUFFER_SIZE = 100
```

Comment:
maybe move this to `common` or `dot/rpc/subscription` or such since `core` isn't using it?

Reply:
moved to `dot/rpc/subscription`